tensorhive/db_seeds.py

```diff
@@ -0,0 +1,70 @@
+from tensorhive.controllers.user.CreateUserController import CreateUserController
```

Comment:
I don't have a clue why we would need to launch whole TensorHive, just to get real GPU UUIDs to test on.